imports.go

```diff
@@ -0,0 +1,46 @@
+package cayley
+
+import (
+	"github.com/google/cayley/graph"
+	_ "github.com/google/cayley/graph/memstore"
+	"github.com/google/cayley/graph/path"
+	"github.com/google/cayley/quad"
+	_ "github.com/google/cayley/writer"
+)
+
+type Iterator graph.Iterator
+type QuadStore graph.QuadStore
+type QuadWriter graph.QuadWriter
+
+type Path path.Path
+
+var StartMorphism = path.StartMorphism
+var StartPath = path.StartPath
+
+var RawNext = graph.Next
+
+type Handle struct {
+	graph.QuadStore
+	graph.QuadWriter
+}
+
+func Quad(subject, predicate, object, label string) quad.Quad {
+	return quad.Quad{subject, predicate, object, label}
+}
+
+func NewMemoryGraph() (*Handle, error) {
+	qs, err := graph.NewQuadStore("memstore", "", nil)
+	if err != nil {
+		return nil, err
+	}
+	qw, err := graph.NewQuadWriter("single", qs, nil)
+	if err != nil {
```

Comment:
I would say so. The close may not be effective depending on the error, but it should be tried. I really must find some time to clean the error handling so we can deal with these things properly.